MovieDB/Manager/CoreDataManager.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  CoreDataManager.swift
+//  MovieDB
+//
+//  Created by le.n.t.trung on 03/11/2022.
+//
+
+import Foundation
+import CoreData
+
+final class PersistentContainer {
+    // MARK: - Core Data stack
+
+    static var persistentContainer: NSPersistentContainer = {
+        let container = NSPersistentContainer(name: "Model")
+        container.loadPersistentStores(completionHandler: { (_, error) in
+            if let error = error as NSError? {
+                fatalError("Unresolved error \(error), \(error.userInfo)")
+            }
+        })
+        return container
+    }()
```

Comment:
thằng `persistentContainer` nên khởi tạo ở trong init
```suggestion
    static let shared =  PersistentContainer()
    private var persistentContainer: NSPersistentContainer?
    
    private init() {
        peristentContainer = NSPersistentContainer(name: "Model")
        peristentContainer.loadPersistentStores(completionHandler: { (_, error) in
            if let error = error as NSError? {
                fatalError("Unresolved error \(error), \(error.userInfo)")
            }
        })
    }
```